components/proxy/src/test/java/com/hotels/styx/StartupConfigTest.java

```diff
@@ -19,14 +19,12 @@
 import org.testng.annotations.BeforeMethod;
 import org.testng.annotations.Test;
 
+import java.io.File;
 import java.net.URISyntaxException;
 import java.nio.file.Path;
 import java.nio.file.Paths;
 
-import static com.hotels.styx.StartupConfig.CONFIG_FILE_LOCATION_VAR_NAME;
-import static com.hotels.styx.StartupConfig.LOGBACK_CONFIG_LOCATION_VAR_NAME;
-import static com.hotels.styx.StartupConfig.STYX_HOME_VAR_NAME;
-import static com.hotels.styx.StartupConfig.newStartupConfigBuilder;
+import static com.hotels.styx.StartupConfig.*;
```

Comment:
We don't use the `*` import in Styx as we like to only import exactly what we are using.

Reply:
As discussed, that's due to IntelliJ settings. The IDE itself was changing the imports as per the "count to use (static) import with *"  If you could do an export of the Settings -> Editor -> Code Style to verify that I don't have any other settings configured that break  codestyle's expectations, that would be great.